src/docfx/build/legacy/files/LegacyResource.cs

```diff
@@ -19,11 +19,23 @@ internal static class LegacyResource
             var legacyManifestOutput = legacyManifestItem.Output;
             var metadata = metadataProvider.GetMetadata(doc);
             metadata = LegacyMetadata.GenerataCommonMetadata(metadata, docset);
+
+            var metadataNeedToBeRemove = new List<string> { "__global" };
+            foreach (var property in metadata)
```

Comment:
use `for` loop to remove and iterate at the same time?

Reply:
not related

Reply:
cannot access jobject key by index

---
In reply to: [240453204](https://github.com/dotnet/docfx/pull/3751#discussion_r240453204) [](ancestors = 240453204)